Documentation/GeneralConventions/ReadmeFile.rst

```diff
@@ -0,0 +1,95 @@
+
+.. index:: File structure; README.rst, README.rst
+.. _readme-rst:
+.. _about-file:
+
+=======================================
+:file:`README.rst` or :file:`README.md`
+=======================================
+
+Full documentation contains both a README.rst and a Documentation/Index.rst
+file. To avoid redundancy in both places, the README.rst in this case usually
+contains only a summary and links to all aspects of the project, i.e. the VCS
+repository, the published documentation and - if available - the TYPO3 Extension
+Repository (TER) page to guide the reader to the next steps. This could be for
```

Comment:
Maybe use a list. Not only the mentioned points, but I would also point to the translation URL (Crowdin) to promote it better.

Reply:
As written in the title I only moved the existing contents for now. I think they will need another round of working on them. It was only the goal to get it out of the file, where it does not belong for now.